src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/KeyboardEventSender.cs

```diff
@@ -24,83 +25,101 @@ public static void Reset()
          */
         public static void SendKeysInOneEvent(int replaySegment, List<(Key, KeyState)> keyStates)
```

Comment:
@RG-nAmKcAz I've made two changes to the SendKeysInOneEvent method (since I'm now using it to simulate key events from the action manager):
1. Also have it send key events through the legacy input manager (added at the top)
2. I noticed that it was not releasing the NativeArray allocated from `DeltaStateEvent.From`, so I added some code to Dispose it after the event is sent.